integr8ly/integreatly-operator
# MGDAPI-3192

Improvements to how the serviceUpdates are updated in the cro strategy config map
to prevent the config map flipping alphabetically causing the cro reconciler to go
in progress incorrectly.

Also some clean up on imports and use of strings from cro packages
Removal of some unused cro functions

# Issue link
https://issues.redhat.com/browse/MGDAPI-3192

# What
See above

# Verification steps

Using CCS Cluster

For quick local verification 
Run `INSTALLATION_TYPE=managed-api USE_CLUSTER_STORAGE=false make cluster/prepare/local`
And  `INSTALLATION_TYPE=managed-api USE_CLUSTER_STORAGE=false make code/run`

Verify that the config map is not flipping back and forth as it described in the ticket. 

For full upgrade verification which is recommended follow steps in -> https://github.com/integr8ly/integreatly-operator/pull/2291
